tests/integration/golang/fixtures/project.go

```diff
@@ -0,0 +1,89 @@
+package fixtures
+
+import (
+	"context"
+	"time"
+
+	"github.com/rotisserie/eris"
+
+	"github.com/G-Research/fasttrackml/pkg/api/aim/response"
+	"github.com/G-Research/fasttrackml/pkg/database"
+)
+
+// ProjectFixtures represents data fixtures object.
+type ProjectFixtures struct {
+	baseFixtures
+}
+
+// NewProjectFixtures creates new instance of ProjectFixtures.
+func NewProjectFixtures(databaseDSN string) (*ProjectFixtures, error) {
+	db, err := database.MakeDBProvider(
+		databaseDSN,
+		1*time.Second,
+		20,
+		false,
+		false,
+		"",
+	)
+	if err != nil {
+		return nil, eris.Wrap(err, "error connection to database")
+	}
+	return &ProjectFixtures{
+		baseFixtures: baseFixtures{db: db.GormDB()},
+	}, nil
+}
+
+func (f *ProjectFixtures) GetProject(ctx context.Context) *response.GetProject {
```

Comment:
can we please add comments for all public methods? it will be required when `linter` will be enabled by default, so let start doing that right now.